blog/schema.sql

```diff
@@ -0,0 +1,11 @@
+create table users(
+    id serial primary key,
+    username text,
+    name text
```

Comment:
I like that you broke this out into different fields (so that a `username` can be unique, but the `name` field could be whatever they want displayed to the public).